### ISSUE TITLE
Problem with `should_launch_new_window` 

### ISSUE THREAD

**Issue by BiliBraker** [ahrm/sioyek#735]
I have enabled `should_launch_new_window` in `prefs_user.config`, but sometimes it does not open the new window.

An example scenario: 
I have a window opened. I try to open a new document by double clicking on it. Sometimes it opens the new document without a problem, but sometimes the new document flashes for a split second and then disappears.
After checking `htop`, I can see the new instance running but not with the new document.

I am using the Appimage and version 2.0.0. on Ubuntu LTS 20.04 with i3wm.

My `prefs_user.config`:
```
background_color 0.0 0.0 0.0
should_launch_new_window 1
super_fast_search 1
```

**Comment by ahrm** [comment]
Are you trying to open a document that already has another sioyek window?

**Comment by BiliBraker** [comment]
No, I am trying to open a separate document.

**Comment by ioannis-vm** [comment]
I am having the same issue, and it's very annoying.
Arch Linux, installed from the aur, my `perfs_user.config` is in `~/.config/sioyek/`,
```
should_launch_new_instance 1
should_launch_new_window 1
```
I would appreciate some help resolving this.

**Comment by ahrm** [comment]
Can you reliably reproduce the issue or is it random?

**Comment by ioannis-vm** [comment]
No, it isn't random. Each time I open a new file I get the current instance of _sioyek_ to replace the document that was being shown with the new one. I have never managed to get it to open the second file in a new window, despite the configuration I showed earlier.

**Comment by ahrm** [comment]
Can't reproduce the issue. Are you using the official appimage builds in release page here: https://github.com/ahrm/sioyek/releases/tag/v2.0.0 or some other package? If the latter, you might want to contact the maintainer of your package.

**Comment by ioannis-vm** [comment]
Apologies, my mistake. I mistyped the name of the config file.
It now works as expected.

**Comment by BiliBraker** [comment]
I recorded my original problem (sorry, it's potato):
https://github.com/ahrm/sioyek/assets/51168364/97710ed7-e6b2-4160-9a81-17de1890523c

**Comment by ahrm** [comment]
Is the issue specific to this GPTScore pdf file or can it happen to other files? If so, can it happen to every file or there are only specific files where this issue happens?

**Comment by BiliBraker** [comment]
I tried it with many files and unfortunately, it can happen to any file randomly.
